clients/src/main/java/org/apache/kafka/common/cache/LRUCache.java

```diff
@@ -29,7 +29,7 @@ public LRUCache(final int maxSize) {
         cache = new LinkedHashMap<K, V>(16, .75f, true) {
             @Override
             protected boolean removeEldestEntry(Map.Entry<K, V> eldest) {
-                return size() > maxSize;
+                return this.size() > maxSize; // require this. prefix to make lgtm.com happy
```

Comment:
nit: this is not only to make lgtm happy right? We can at least save one virtual call in java byte code (assuming JIT is not yet that smart :P)

Reply:
How is this saving a virtual call?